common/src/features/Events/utils/parseCalendarEvent.ts

```
@@ -265,13 +266,20 @@ function parseAlarms(valarms?: VCalComponent[]): Valarms | undefined {
           alarm.description = safeString(value)
           break
         case 'attendee':
-          alarm.attendee = userAttendee.fromEmailField(safeString(value))
+          const attendee = userAttendee.fromEmailField(safeString(value))
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)